lib/index.js

```diff
@@ -15,6 +19,7 @@ var Appboy = module.exports = integration('Appboy')
   .option('apiKey', '')
   .option('safariWebsitePushId', '')
   .option('automaticallyDisplayMessages', true)
+  .tag('<script src="https://js.appboycdn.com/web-sdk/1.6/appboy.min.css">')
```

Comment:
What's the need to add a css file? Might be worth a quick comment or link to their docs.